csp/utils.py

```diff
@@ -53,9 +61,20 @@ def build_policy(config=None, update=None, replace=None):
                 csp[k] += tuple(v)
 
     report_uri = csp.pop('report-uri', None)
+    upgrade_insecure_requests = csp.pop('upgrade-insecure-requests', None)
+    block_all_mixed_content = csp.pop('block-all-mixed-content', None)
+
     policy = ['%s %s' % (kk, ' '.join(vv)) for kk, vv in
               csp.items() if vv is not None]
+
     if report_uri:
         report_uri = map(force_text, report_uri)
         policy.append('report-uri %s' % ' '.join(report_uri))
+
+    if upgrade_insecure_requests:
+        policy.append('upgrade-insecure-requests ')
+
+    if block_all_mixed_content:
+        policy.append('block-all-mixed-content ')
```

Comment:
👍 yeah that's nicer and lets us handle future boolean CSP directives

I'll drop the trailing space too, since the join at the end adds spaces.